src/script/ceph-release-notes

```diff
@@ -38,9 +38,9 @@ from git import Repo
 fixes_re = re.compile(r"Fixes\:? #(\d+)")
 reviewed_by_re = re.compile(r"Rev(.*)By", re.IGNORECASE)
 # labels is the list of relevant labels defined for github.com/ceph/ceph
-labels = ['bluestore', 'build/ops', 'cephfs', 'common', 'core', 'mgr',
+labels = set(['bluestore', 'build/ops', 'cephfs', 'common', 'core', 'mgr',
```

Comment:
Actually, if I'm not mistaken, the script requires py3 as of e194926301db3e0bb3f9a062be01a70c3c68246e